lib/single-new-eks-awsnative-fargate-observability-construct/index.ts

```diff
@@ -0,0 +1,70 @@
+import { Construct } from 'constructs';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import { ObservabilityBuilder } from '../common/observability-builder';
+import * as eks from 'aws-cdk-lib/aws-eks';
+
+
+export default class SingleNewEksAWSNativeFargateobservabilityConstruct {
+    constructor(scope: Construct, id: string) {
+        // AddOns for the cluster
+        const stackId = `${id}-observability-accelerator`;
+
+        const account = process.env.COA_ACCOUNT_ID! || process.env.CDK_DEFAULT_ACCOUNT!;
+        const region = process.env.COA_AWS_REGION! || process.env.CDK_DEFAULT_REGION!;
+
+        const cloudWatchAdotAddOn = new blueprints.addons.CloudWatchAdotAddOn({
+            deploymentMode: blueprints.cloudWatchDeploymentMode.DEPLOYMENT,
+            namespace: 'default',
+            name: 'adot-collector-cloudwatch',
+            metricsNameSelectors: ['apiserver_request_.*', 'container_memory_.*', 'container_threads', 'otelcol_process_.*'],
+        });
+        
+        const addOns: Array<blueprints.ClusterAddOn> = [
+            new blueprints.addons.VpcCniAddOn(),
+            new blueprints.addons.CoreDnsAddOn({
+                version:"v1.10.1-eksbuild.1",
+                configurationValues:{ computeType: "Fargate" }
+            }),
+            new blueprints.addons.MetricsServerAddOn(),
+            new blueprints.addons.KubeStateMetricsAddOn(),
+            new blueprints.addons.KubeProxyAddOn(),
+            new blueprints.addons.AwsLoadBalancerControllerAddOn(),
+            new blueprints.addons.CertManagerAddOn({
+                installCRDs:true,
+                createNamespace:true,
```

Comment:
first 3 parameters are already set by addon by default so not required to pass.  Is there a reason why you need Vaules, is that needed for Fargate. Reason why im asking this is because these values should be passed as propos to builder in ObservabilityBuilder as blueprints and we can use that only after next release.